junction/src/main/java/org/killbill/billing/junction/plumbing/billing/DefaultInternalBillingApi.java

```diff
@@ -151,15 +154,22 @@ private void addBillingEventsForBundles(final List<SubscriptionBaseBundle> bundl
 
         }
 
+        final List<Tag> bundleTags = getTagsForObjectType(ObjectType.BUNDLE, tags);
```

Comment:
⚠️ You need to filter by `bundleId` too (see original call below), because each bundle could be independently tagged with `AUTO_INVOICING_OFF`.